docs/ruleset_functions/get_heat_rejection_loops_connected_to_baseline_systems.md

```diff
@@ -0,0 +1,46 @@
+
+## get_heat_rejection_loops_connected_to_baseline_systems
+
+Description: Get a list of all heat rejection loops in an RMI that are connected to a baseline HVAC System (Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11.1b, 12b)
+
+Inputs:  
+- **RMI**: RMI that needs to get the list of primary and secondary loops (usually baseline RMI).
+
+Returns: 
+- **heat_rejection_loop_list**: A list that saves the ids of heat rejection loops in the model that serve baseline system types, e.g. [heat_rejection_loop1.id, heat_rejection_loop2.id]. If RMI does not have any qualifying heat rejection loops, it will return an empty list ([]).
+
```

Comment:
@KarenWGard would it make sense to use https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/get_primary_secondary_loops_dict.md to identify P/S loops and then identify the heat rejection system serving the primary loop?

Reply:
`get_primariy_seciondary_loops_dict` returns only loops that are correctly formatted as primary / secondary loops.  The way I've written this code, the user will get feedback about heat rejection equipment regardless of whether they modeled the primary / secondary loops correctly.  As a modeler, I prefer to resolve as many issues a possible in each iteration of modeling, so it was a personal choice to do it this way.

Reply:
updated the function per our notes on the call.  The function now relies on `get_primary_secondary_loops_dict` to maintain consistency with other rules